src/test/groovy/com/commercehub/gradle/plugin/avro/AvroPluginFunctionalSpec.groovy

```diff
@@ -59,16 +59,24 @@ class AvroPluginFunctionalSpec extends FunctionalSpec {
 
         when:
         def result = run()
+        def interopJavaContent = projectFile("build/generated-main-avro-java/org/apache/avro/Interop.java").text
 
         then:
         result.task(":generateAvroProtocol").outcome == SUCCESS
         result.task(":generateAvroJava").outcome == SUCCESS
         result.task(":compileJava").outcome == SUCCESS
+        projectFile("build/generated-main-avro-java/org/apache/avro/Interop.java").file
         projectFile(buildOutputClassPath("org/apache/avro/Foo.class")).file
         projectFile(buildOutputClassPath("org/apache/avro/Interop.class")).file
         projectFile(buildOutputClassPath("org/apache/avro/Kind.class")).file
         projectFile(buildOutputClassPath("org/apache/avro/MD5.class")).file
         projectFile(buildOutputClassPath("org/apache/avro/Node.class")).file
+        interopJavaContent
```

Comment:
This is a good addition.  Thank you for suggesting it.  Do you think there's value in also adding assertions for the more "normal" fields, or would that be overkill?

Reply:
Done. For all datatypes ( which I know ).